lib/src/components/NxDropdown/NxDropdown.scss

```diff
@@ -26,7 +26,9 @@ $nx-dropdown-width: 250px;
   width: $nx-dropdown-width;
 
   .nx-dropdown__toggle-label {
+    @include truncate-ellipsis();
     flex-grow: 1;
+    max-width: $nx-dropdown-width;
```

Comment:
This doesn't seem necessary or correct.  The parent already has this width, and contains another element (the caret) in addition to this element, so the apparent max here is actually 198px.  But it doesn't seem like that needs to be made explicit, it works without it.

Reply:
> This doesn't seem necessary or correct. The parent already has this width, and contains another element (the caret) in addition to this element, so the apparent max here is actually 198px. But it doesn't seem like that needs to be made explicit, it works without it.

https://github.com/sonatype/sonatype-react-shared-components/pull/59/commits/addb08d80953eeea53853937acaf642e93314baa